app/views/admin/schools/index.html.erb

```diff
@@ -15,26 +15,43 @@
   <% end %>
 </div>
 
-<table class="govuk-table">
-  <thead>
-    <tr class="govuk-table__row">
-      <th scope="col" class="govuk-table__header">School</th>
-      <th scope="col" class="govuk-table__header">URN</th>
-      <th scope="col" class="govuk-table__header">Local Authority</th>
-      <th scope="col" class="govuk-table__header">Group/network</th>
-      <th scope="col" class="govuk-table__header">Induction tutor</th>
-    </tr>
-  </thead>
-  <tbody class="govuk-table__body">
-    <% @schools.each do |school| %>
+<div class="nhsuk-table-container">
+  <table class="govuk-table nhsuk-table-responsive">
+    <thead class="nhsuk-table__head">
       <tr class="govuk-table__row">
-        <td class="govuk-table__cell"><%= govuk_link_to school.name, admin_school_path(school) %></td>
```

Comment:
Ditto here, does this want `nhsuk-table__row`?

Reply:
same as above (even though github has put your comment in a weird place - i think you're talking about the line on the other side of the diff?)